pkg/gen/python-templates/helm/v2/helm.py

```diff
@@ -0,0 +1,427 @@
+
+            # Use 'helm template' to create a combined YAML manifest.
+            cmd = ['helm', 'template', chart, '--name', release_name,
+                   '--values', default_values, '--values', overrides.name]
+            cmd.extend(namespace_arg)
+
+            output = subprocess.run(
+                cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True, check=True)
+            yaml_str: str = output.stdout
+
+            # Parse the manifest and create the specified resources.
+            return _parse_yaml_document(yaml.safe_load_all(yaml_str), opts, config.transformations)
+
+
+def _fetch(chart: str, opts: FetchOpts) -> None:
+    cmd: List[str] = ['helm', 'fetch', chart]
+
+    # Untar by default.
+    if opts.untar is not False:
+        cmd.append('--untar')
+
+    if opts.version:
+        cmd.extend(['--version', opts.version])
+    if opts.ca_file:
+        cmd.extend(['--ca-file', opts.ca_file])
+    if opts.cert_file:
+        cmd.extend(['--cert-file', opts.cert_file])
+    if opts.key_file:
+        cmd.extend(['--key-file', opts.key_file])
+    if opts.destination:
+        cmd.extend(['--destination', opts.destination])
+    if opts.keyring:
+        cmd.extend(['--keyring', opts.keyring])
+    if opts.password:
+        cmd.extend(['--password', opts.password])
+    if opts.repo:
+        cmd.extend(['--repo', opts.repo])
+    if opts.untar_dir:
+        cmd.extend(['--untardir', opts.untar_dir])
+    if opts.username:
+        cmd.extend(['--username', opts.username])
+    if opts.home:
+        cmd.extend(['--home', opts.home])
+    if opts.devel:
+        cmd.append('--devel')
+    if opts.prov:
+        cmd.append('--prov')
+    if opts.verify:
+        cmd.append('--verify')
+
+    subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True, check=True)
+
+
+class Chart(pulumi.ComponentResource):
+    """
+    Chart is a component representing a collection of resources described by an arbitrary Helm
+    Chart. The Chart can be fetched from any source that is accessible to the `helm` command
+    line. Values in the `values.yml` file can be overridden using `ChartOpts.values` (equivalent
+    to `--set` or having multiple `values.yml` files). Objects can be transformed arbitrarily by
+    supplying callbacks to `ChartOpts.transformations`.
+
+    `Chart` does not use Tiller. The Chart specified is copied and expanded locally; any values
+    that would be retrieved in-cluster would be assigned fake values, and none of Tiller's
+    server-side validity testing is executed.
+
+    The semantics of `update` on a Chart are identical to those of Helm and kubectl; for example,
+    unlike a "normal" Pulumi program, updating a ConfigMap does not trigger a cascading update
+    among Deployments that reference it.
+
+    NOTE: `Chart` will attempt to sort the resources in the same way that Helm does, to ensure
```

Comment:
This comment isn't true, right? I recall we did some limited sorting in the Node Helm implementation but I thought we ended up not doing that in lieu of having unlimited parallelism in the engine.

Reply:
Good catch. I didn't bother with that for the Python implementation, and seems to be working fine.